README.md

```diff
@@ -102,10 +102,11 @@ allows you to commit to your repository without triggering a new version.
 ### `in`: Clone the repository, at the given ref.
 
 Clones the repository to the destination, and locks it down to a given ref.
-Returns the resulting ref as the version.
 
-Submodules are initialized and updated recursively.
+It will return the same ref as version, unless ref is `HEAD`, which
```

Comment:
The only case is that no version is passed to `in`, which in that case it checkouts the latest commit of the branch, but indeed it is not expressed properly. Let me rephrase it.